src/com/lftechnology/java/training/sanish/application/component/PageMenu.java

```diff
@@ -0,0 +1,139 @@
+package com.lftechnology.java.training.sanish.application.component;
+
+import com.lftechnology.java.training.sanish.application.controller.EmployeeController;
+import com.lftechnology.java.training.sanish.application.controller.LoginController;
+import com.lftechnology.java.training.sanish.application.model.domain.Employee;
+
+/**
+ * Page menu options
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class PageMenu {
+    private PageMenu() {
+    }
+
+    /**
+     * Dashboard page menu options
+     *
+     * @param menuSelected {@link Integer}
+     * @param employee     {@link Employee}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void dashboardMenu(int menuSelected, Employee employee) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "dashboardMenu" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)